aws/resource_aws_glue_connection_test.go

```diff
@@ -426,6 +426,7 @@ resource "aws_glue_connection" "test" {
   name = "%[1]s"
 
   physical_connection_requirements {
+    availability_zone      = "${aws_availability_zones.available[0].id}"
```

Comment:
This addition was not working in the acceptance testing:

```
--- FAIL: TestAccAWSGlueConnection_PhysicalConnectionRequirements (0.91s)
	testing.go:518: Step 0 error: Error loading configuration: Error loading /var/folders/v0/_d108fkx1pbbg4_sh864_7740000gn/T/tf-test582550071/main.tf: Error reading config for aws_glue_connection[test]: parse error at 1:38: expected "}" but found "."
```

Changing it to `availability_zone      = "${aws_subnet.test.0.availability_zone}"` worked as expected 👍 

Also we normally recommend adding a check that a new attribute is at least set in the acceptance testing: `resource.TestCheckResourceAttrSet(resourceName, "physical_connection_requirements.0.availability_zone"),`

Reply:
OK excellent thanks for your comments and effort! I'll check this `TestCheckResourceAttrSet` further later ;)